File: 1293-three-consecutive-odds/three-consecutive-odds.go
func threeConsecutiveOdds(arr []int) bool {
    odd := 0;
    for _, num := range arr {
        if num % 2 != 0 {
            odd++;
            if odd == 3 {
                return true;
            }
        } else {
            odd = 0;
        }
    }

    return false;
}

File: 1293-three-consecutive-odds/three-consecutive-odds.java
class Solution {
    public boolean threeConsecutiveOdds(int[] arr) {
        int odd = 0;
        for (int num : arr) {
            if (odd == 3) return true;
            odd++;
            if (num % 2 == 0) odd = 0;
        }

        if (odd == 3) return true;

        return false;
    }
}